test/unit/dropin.js

```diff
@@ -186,6 +187,16 @@ describe('Dropin', function () {
       }.bind(this));
     });
 
+    it('injects stylesheet with correct id', function (done) {
```

Comment:
Do we have a test already that says we don't inject the stylesheet if a stylesheet with that id already exists?